lib/routes/account.js

```diff
@@ -34,6 +38,9 @@ module.exports = function (
   metricsContext
   ) {
 
+  var schemaPath = path.resolve(__dirname, PUSH_PAYLOADS_SCHEMA_PATH)
```

Comment:
@eoger could you please a doc to this code, explaining what it is doing here. aka `// loads the JSON schema for ...`